Source/WebCore/Sources.txt

```diff
@@ -2403,9 +2403,11 @@ platform/graphics/DisplayRefreshMonitorClient.cpp
 platform/graphics/DisplayRefreshMonitorManager.cpp
 platform/graphics/DisplayUpdate.cpp
 platform/graphics/FloatLine.cpp
+platform/graphics/FloatPlane3D.cpp
 platform/graphics/FloatPoint.cpp
 platform/graphics/FloatPoint3D.cpp
 platform/graphics/FloatPolygon.cpp
+platform/graphics/FloatPolygon3D.cpp
```

Comment:
FloatPlane3D and FloatPolygon3D are used only by TextureMapper. Cocoa port won't use it because it uses CoreAnimation. I think these files should be moved to platform/graphics/texmap.
Or, enclosed .cpp and .h by `#if USE(TEXTURE_MAPPER)` to express that this is only for TextureMapper.

Reply:
Moved to platform/graphics/texmap